python_modules/libraries/dagster-dbt/dagster_dbt/cli/app.py

```diff
@@ -89,6 +90,9 @@ def copy_scaffold(
         for target in profile["outputs"].values()
     ]
 
+    if use_dbt_project_package_data_dir:
```

Comment:
I think this should just use the passed in dbt_project_dir and just error if the dbt_project_dir is not under the start path